alfresco-java-rest-api/alfresco-java-rest-api-lib/alfresco-event-gateway-api/docs/JsonNode.md

```diff
@@ -0,0 +1,5 @@
+# JsonNode
```

Comment:
Is this JSON node needed? I think you deleted before

Reply:
Shoot, that went under my radar, thanks